### PR TITLE
Reference page for gsctl scale cluster

### PR DESCRIPTION
**Pull request by marians**
The title says it

**To be merged once the API function is available and a new gsctl version is released.**

### CODE REVIEW

File: content/reference/gsctl/scale-cluster.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+title = "gsctl Command Reference: scale cluster"
+description = "The 'gsctl scale cluster' command allows to add or remove worker nodes to reach a desired number."
+date = "2017-08-01"
+type = "page"
+weight = 41
++++
+
+# `gsctl scale cluster`
+
+The `gsctl scale cluster` command allows to scale a cluster, i. e. to add or remove worker nodes to/from a cluster.
```
Comment (rossf7) [inline content/reference/gsctl/scale-cluster.md]: "allows you to scale" and "i.e."

File: content/reference/gsctl/scale-cluster.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+When adding worker nodes, no confirmation is required.
+
+After the command execution is finished, it can take a couple of minutes until the new workers are available in your Kubernetes clusters.
+
+## Full Argument Reference {#arguments}
+
+- `-w`, `--num-workers` (required): The desired amount of worker nodes after scaling.
+- `--force`: If set, no confirmation is required when reducing the number of workers. You should only use this in automations when you are sure that reducing the number of workers is desired.
+- `--memory-gb`: Currently not functional. Once scaling is supported on KVM, which argument will allow to configure the worker node(s) to be added.
```
Comment (rossf7) [inline content/reference/gsctl/scale-cluster.md]: "this argument"

Comment (puja108) [inline content/reference/gsctl/scale-cluster.md]: here and below, maybe we should add what you are configuring (i.e. memory, storage, cpus)

**Comment by marians** [comment]
Thanks for the reviews!

**Comment by oponder** [comment]
Nice on the clarification. LGTM

File: content/reference/gsctl/scale-cluster.md [hunk 3]
```diff
@@ -14,10 +14,11 @@ When using the command, you define how many worker nodes (short: workers) you in
 
 When **scaling up** (increasing the number of workers), the new workers will be configured in the same way as the existing workers.
 
+When **scaling down** (reducing the number of workers), the decision which workers get removed is left to the AWS Auto Scaling Groups logic. We use the [default termination policy](http://docs.aws.amazon.com/autoscaling/latest/userguide/as-instance-termination.html#default-termination-policy). In brief, this policy will remove the oldest workers first. If all workers have the same age, the workers to be removed will be selected at random.
```
Comment (rossf7) [inline content/reference/gsctl/scale-cluster.md]: "decision on which"

**Pull request merged**